ops/jenkins/bfd-build-deploy/Jenkinsfile

```diff
@@ -36,56 +90,56 @@ spec:
-        defaultValue: false)
-
-        booleanParam(name: 'force_build_docker_host_ami',
-        description: 'When true, the docker host AMI based on branch will be built. Note: `force_build_amis` must' +
-                'also be true.',
-        defaultValue: false)
-
-        string(
-        name: 'env',
-        description: 'The BFD Environment to deploy apps to'
-        )
-
-        string(
-        name: 'migrator_ami_override',
-        description: 'The AMI ID to deploy migrator from',
-        defaultValue: null
-        )
-
-        string(
-        name: 'pipeline_ami_override',
-        description: 'The AMI ID to deploy pipeline from',
-        defaultValue: null
-        )
-
-        string(
-        name: 'server_ami_override',
-        description: 'The AMI ID to deploy server from',
-        defaultValue: null
-        )
-
-
-        booleanParam(
-        name: 'force_migrator_deployment',
-        description: 'When true, force the migrator to deploy.',
-        defaultValue: false
-        )
-
-        string(
-        name: 'server_regression_image_override',
-        description: 'Overrides the Docker image tag used when deploying the server-regression lambda',
-        defaultValue: null
-        )
-  }
+//   parameters {
+//         booleanParam(name: 'verbose_mvn_logging',
+//         description: 'When true, `mvn` will produce verbose logs.',
+//         defaultValue: false)
+//
+//         booleanParam(name: 'force_build_amis',
+//         description: 'When true, AMIs based on branch will be built.',
+//         defaultValue: false)
+//
+//         booleanParam(name: 'force_build_docker_host_ami',
+//         description: 'When true, the docker host AMI based on branch will be built. Note: `force_build_amis` must' +
+//                 'also be true.',
+//         defaultValue: false)
+//
+//         string(
+//         name: 'env',
+//         description: 'The BFD Environment to deploy apps to'
+//         )
+//
+//         string(
+//         name: 'migrator_ami_override',
+//         description: 'The AMI ID to deploy migrator from',
+//         defaultValue: null
+//         )
+//
+//         string(
+//         name: 'pipeline_ami_override',
+//         description: 'The AMI ID to deploy pipeline from',
+//         defaultValue: null
+//         )
+//
+//         string(
+//         name: 'server_ami_override',
+//         description: 'The AMI ID to deploy server from',
+//         defaultValue: null
+//         )
+//
+//
+//         booleanParam(
+//         name: 'force_migrator_deployment',
+//         description: 'When true, force the migrator to deploy.',
+//         defaultValue: false
+//         )
+//
+//         string(
+//         name: 'server_regression_image_override',
+//         description: 'Overrides the Docker image tag used when deploying the server-regression lambda',
+//         defaultValue: null
+//         )
+//   }
```

Comment:
Probably safe to delete this.

Reply:
I have removed all of the commented out parameter sections.